addon/components/polaris-drop-zone.js

```diff
@@ -372,10 +374,11 @@ export default Component.extend(ContextBoundEventListenersMixin, {
 
     this.dragTargets = [];
 
-    this.get('state').setProperties({
+    state.setProperties({
       dragging: false,
       error: rejectedFiles.length > 0,
     });
+    state.incrementProperty('numFiles', acceptedFiles.length);
```

Comment:
nit: should we just move in the `setProperties` above instead of using `incrementProperty`?